lib/block-supports/block-style-variations.php

```diff
@@ -334,7 +334,25 @@ function gutenberg_resolve_block_style_variations_from_theme_json_partials( $the
  *
  * @return WP_Theme_JSON_Data_Gutenberg
  */
-function gutenberg_resolve_block_style_variations_from_primary_theme_json( $theme_json ) {
+function gutenberg_resolve_block_style_variations_from_theme_json_default( $theme_json ) {
+	$theme_json_data        = $theme_json->get_data();
+	$block_style_variations = $theme_json_data['styles']['blocks']['variations'] ?? array();
+	$variations_data        = gutenberg_resolve_block_style_variations( $block_style_variations );
+
+	return gutenberg_merge_block_style_variations_data( $variations_data, $theme_json, 'default' );
+}
+
+/**
+ * Merges shared block style variations registered within the
+ * `styles.blocks.variations` property of the primary theme.json file.
```

Comment:
If we needed to rename the functions for clarity, perhaps the reference to "primary theme.json file" here also needs updating?